common/src/platform/linux/time.c

```diff
@@ -0,0 +1,104 @@
+/*
+Looking Glass - KVM FrameRelay (KVMFR) Client
+Copyright (C) 2017-2020 Geoffrey McRae <geoff@hostfission.com>
```

Comment:
I didn't author this, please give yourself credit :)

Reply:
Thanks! :)